arithmetization/src/test/java/net/consensys/linea/zktracer/testing/BytecodeRunner.java

```diff
@@ -37,6 +38,7 @@
 @Accessors(fluent = true)
 public final class BytecodeRunner {
   private final Bytes byteCode;
+  ToyExecutionEnvironment toyExecutionEnvironment;
```

Comment:
Why is this field needed?

Reply:
Sorry, I got confused and believed this was the oob PR. Anyway, I may do something similar for ecdata and at some point I need that feature in `ByteCodeRunner`.